object-mapper.rst

```diff
@@ -0,0 +1,259 @@
+Object Mapping
+==============
+
+Symfony provides a mapper to transform a given object to another one.
+
+.. _activating_the_serializer:
+
+Installation
+------------
+
+Run this command to install the ``object-mapper`` before using it:
+
+.. code-block:: terminal
+
+    $ composer require symfony/object-mapper
+
+Using the ObjectMapper Service
+------------------------------
+
+Once enabled, the object mapper service can be injected in any service where
```

Comment:
Once installed ?

because writing enabled can mean that it is not enabled:usable by default after installation